sklearn/gaussian_process/_gpc.py

```diff
@@ -648,19 +652,20 @@ def __init__(
         self.n_jobs = n_jobs
 
     def fit(self, X, y):
-        """Fit Gaussian process classification model
+        """Fit Gaussian process classification model.
 
         Parameters
         ----------
         X : array-like of shape (n_samples, n_features) or list of object
             Feature vectors or other representations of training data.
 
         y : array-like of shape (n_samples,)
-            Target values, must be binary
+            Target values, must be binary.
 
         Returns
         -------
-        self : returns an instance of self.
+        self : object
```

Comment:
```suggestion
        self
```

Reply:
I would let `self : object` for consistency with all other docstring in the package. We could make another PR to remove the type later on.